tests/e2e/workloads/test_couchbase.py

```diff
@@ -236,6 +236,11 @@ def teardown(self):
             adm_data = templating.load_yaml(adm_yaml)
             adm_obj = OCS(**adm_data)
             adm_obj.delete()
+        nsinfo = self.pod_obj.exec_oc_cmd(command="get namespace")
```

Comment:
We should add this in pillowfight cleanup routine

Reply:
I moved this change to the end of the pillowfight cleanup.